pkg/nimble/autoadv/include/nimble_autoadv.h

```diff
@@ -35,25 +35,25 @@ extern "C" {
 #endif
 
 /**
-* @brief    Name of the device for the advertising procedure. If this is not
+ * @brief    Name of the device for the advertising procedure. If this is not
  *          defined, it will be defined as NULL, resulting in not configuring
  *          a name at all.
-*/
-#ifndef NIMBLE_AUTOADV_DEVICE_NAME
-    #define NIMBLE_AUTOADV_DEVICE_NAME NULL
+ */
+#ifndef CONFIG_NIMBLE_AUTOADV_DEVICE_NAME
+#define CONFIG_NIMBLE_AUTOADV_DEVICE_NAME       NULL
```

Comment:
What about a different default for it like "nimble on RIOT OS" or something? This would make it easier for people new to it, e.g. when trying out `gnrc_networking`, to discover their device.